cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4369,6 +4381,159 @@ msgstr "Chwiliwch am gyfryngwr teulu yn eich ardal chi i ganfod a ydych yn gymwy
+msgid "Disproportionate burden"
+msgstr "Baich anghymesur"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Not applicable"
+msgstr "Ddim yn berthnasol"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Content that’s not within the scope of the accessibility regulations"
+msgstr "Cynnwys nad yw o fewn cwmpas y rheoliadau hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "How we tested this service"
+msgstr "Sut y buom yn profi’r gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "July"
+msgstr "Gorffennaf"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "September"
+msgstr "Medi"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "December"
+msgstr "Rhagfyr"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "This service was last tested on %(day)s %(month)s %(year)s."
+msgstr "Cafodd y gwasanaeth hwn ei brofi ddiwethaf ar %(day)s %(month)s %(year)s."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The test was carried out by the Digital Accessibility Centre (<abbr>DAC</abbr>)."
+msgstr "Cynhaliwyd y prawf gan y Ganolfan Hygyrchedd Digidol (<abbr>DAC</abbr>)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The <abbr title='Digital Accessibility Centre'>DAC</abbr> tested the service available at <a class='govuk-link' href='https://checklegalaid.service.gov.uk/'>https://checklegalaid.service.gov.uk/</a>."
+msgstr "Profodd y <abbr title='Ganolfan Hygyrchedd Digidol'>DAC</abbr> y gwasanaeth sydd ar gael yn <a class='govuk-link' href='https://www.gov.uk/gwirio-os-ydych-yn-gymwys-i-gael-cymorth-cyfreithiol'>https://checklegalaid.service.gov.uk/</a>."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "What we’re doing to improve accessibility"
+msgstr "Beth rydyn ni’n ei wneud i wella hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "We are working to fix the issues listed in this statement by %(month)s %(year)s. When the issues are fixed we will send the service for a re-test with the <abbr title='Digital Accessibility Centre'>DAC</abbr>."
+msgstr "Rydyn ni’n gweithio i ddatrys y materion a restrir yn y datganiad hwn erbyn mis %(month)s %(year)s. Pan fydd y materion yn cael eu datrys, byddwn yn anfon y gwasanaeth i gael ail brawf gyda’r <abbr title='Ganolfan Hygyrchedd Digidol'>DAC</abbr>."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "We will monitor the accessibility of this service on an ongoing basis and fix any accessibility issues reported to us."
+msgstr "Byddwn yn monitro hygyrchedd y gwasanaeth hwn yn barhaus ac yn datrys unrhyw broblemau hygyrchedd y rhoddir gwybod i ni amdanyn nhw."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "This statement was prepared on %(day)s %(month)s %(year)s."
+msgstr "Cafodd y datganiad hwn ei baratoi ar %(day)s %(month)s %(year)s."
```

Comment:
Line 114 of accessibility statement
Seems to have been replaced by line **4517** of the **messages.po**
`The <abbr title='Digital Accessibility Centre'>DAC</abbr> tested the service available at <a class='govuk-link' href='https://checklegalaid.service.gov.uk/'>https://checklegalaid.service.gov.uk/</a>.`

Reply:
This sentence is still used, probably moved around a bit though.  It is in a 2-sentence paragraph, each with their own translations.